<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTTPFuzzWindow</class>
 <widget class="QMainWindow" name="HTTPFuzzWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP Fuzzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Http Tester</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>931</width>
        <height>531</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Request</string>
       </attribute>
       <widget class="QLineEdit" name="txt_url">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>811</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="txt_request">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>901</width>
          <height>371</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_send">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>440</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QComboBox" name="dropdw_method">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>87</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Response</string>
       </attribute>
       <widget class="QTextEdit" name="txt_response">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>921</width>
          <height>491</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
